avm/res/storage/storage-account/blob-service/main.bicep

```diff
@@ -12,7 +12,7 @@ param automaticSnapshotPolicyEnabled bool = false
 @description('Optional. The blob service properties for change feed events. Indicates whether change feed event logging is enabled for the Blob service.')
 param changeFeedEnabled bool = false
 
-@minValue(0)
+@minValue(1)
 @maxValue(146000)
 @description('Optional. Indicates whether change feed event logging is enabled for the Blob service. Indicates the duration of changeFeed retention in days. A "0" value indicates an infinite retention of the change feed.')
```

Comment:
Please be aware, the parameter's description & new `min` value don't align.

Reply:
I agree, description should be updated to align with the documentation: https://learn.microsoft.com/en-us/azure/templates/microsoft.storage/storageaccounts/blobservices?pivots=deployment-language-bicep#changefeed

Reply:
Description updated